repository/Cormas-Core/CormasModel.class.st

```diff
@@ -575,7 +575,7 @@ CormasModel class >> defaultDelayChangesSetting [
 
 { #category : #'+ simulation settings' }
 CormasModel class >> defaultFinalStep [
-	self subclassResponsibility 
+	^100
```

Comment:
@SergeStinckwich Why do you want to change here? It seems each concrete model defines its preferred number of steps.

Reply:
I think we need a value by default, because every time you build a new model, you have to provide this number. Originally it was ^nil

Reply:
Okay, CMSimManager>>finalTime is the real one and this method gives only the default value of that. And, CMSimManager>>finalTime: is set at the simulation setup code. So, this method is not really the final decision maker. Okay, I'm fine with the proposed change.

Reply:
I will open a new issue to discuss about the interest of having a defaultFinalStep value.

Reply:
👍